exe/etcd-env

```diff
@@ -5,14 +5,18 @@ require 'json'
 require 'net/http'
 
 def usage
-  puts "Usage: #{$PROGRAM_NAME} <etcd key> <command>"
+  puts "Usage: #{$PROGRAM_NAME} <etcd key> [ <etcd key> ... ] -- <command>"
 end
 
-def env_vars(key)
-  response = fetch "http://#{etcd_host}:#{etcd_port}/v2/keys#{key}"
-  (JSON.parse(response)['node']['nodes'] || [])
-    .map { |var| [var['key'].split('/').last, var['value']] }
-    .to_h
+def env_vars(keys)
+  var = Array.new
+  keys.each do |key|
```

Comment:
Instead of creating an empty array and appending to it in a loop, you can use `map`:

```ruby
keys.map do |key|
   ...
end
```